src/cmd/setup-tachyon.js

```diff
@@ -143,17 +157,24 @@ module.exports = class SetupTachyonCommands extends CLICommandBase {
 		}
 	}
 
-	async _loadConfig(options, requiredFields) {
+	async _loadConfig({ options, requiredFields, deviceInfo }) {
 		const configFromFile = await this._loadConfigFromFile(options.loadConfig);
+		const optionsFromDevice = {};
 		const cleanedOptions = Object.fromEntries(
 			// eslint-disable-next-line no-unused-vars
 			Object.entries(options).filter(([_, v]) => v !== undefined)
 		);
+		if (deviceInfo) {
+			optionsFromDevice.region = deviceInfo.region.toLowerCase() !== 'unknown' ? deviceInfo.region : 'NA';
+			optionsFromDevice.board = deviceInfo.osVersion === 'Ubuntu 20.04' ? 'formfactor_dvt' : 'formfactor';
+		}
 		const config = {
 			...this.defaultOptions,
 			...configFromFile,
+			...optionsFromDevice,
```

Comment:
Let's flip the order so it's possible to provide the region and board from the file.
```suggestion
			...optionsFromDevice,
			...configFromFile,
```

Reply:
Done here! https://github.com/particle-iot/particle-cli/pull/825/commits/73615ec11266dc9a22bed4522cdcef5e301a63ae